README.md

```diff
@@ -59,6 +66,58 @@ billing_cycle.previous_due_at
 # => nil
 ```
 
+### Time Elapsed and Remaining
+
+The time elapsed and remaining can be used for displaying how much time has been used
+and how much time is left in a billing cycle.
+
+```ruby
+original_billing_date = Time.zone.parse("2019-06-01 00:00:00")
+billing_interval = 1.month
+billing_cycle = BillingCycle::BillingCycle.new(original_billing_date, billing_interval)
+
+Time.zone.now
+# => Sun, 16 Jun 2019 00:00:00 CDT -05:00
+
+billing_cycle.time_elapsed
+# => 1296000.0
+
+billing_cycle.time_elapsed(1.day, Time.zone.parse("2019-06-07 00:00:00"))
```

Comment:
Add a note that 1.day is asking how many days have elapsed... and from the 7th of the month.